docs/client-openwrt-router-wireguard.md

```diff
@@ -1,4 +1,4 @@
-# Using Router wıth OpenWRT as a Client with WireGuard
+# Using Router with OpenWRT as a Client with WireGuard
```

Comment:
This is interesting, because the original character was U+0131 Latin Small Letter Dotless I